examples/abitest/module_0/rust/src/lib.rs

```diff
@@ -51,6 +51,7 @@ const STORAGE_NAME_PREFIX: &str = "abitest";
 
 const HTTP_ADDR: &str = "[::]:8383";
 const ADDITIONAL_TEST_SERVER_ADDR: &str = "[::]:8081";
+const ADDITIONAL_TEST_SERVER_ADDR_2: &str = "[::]:8082";
```

Comment:
Suggestion for another PR: Would be nice to have the explanation in the commit message as a comment here. I remember that there was a comment earlier about why using different port numbers, and that is why I had changed all the port numbers to the same value.